pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
I think you'd need to use pnpm 8.6 and above to generate the lockfile v6.1

Reply:
Hm, very odd:

> pnpm -v
> 8.6.2

Reply:
Got it.